src/Nethermind/Nethermind.Db.Rocks/ColumnsDb.cs

```diff
@@ -25,20 +27,39 @@ public abstract class ColumnsDb<T> : DbOnTheRocks, IColumnsDb<T>
     {
         private readonly IDictionary<T, IDb> _columnDbs = new Dictionary<T, IDb>();
         
-        protected ColumnsDb(string basePath, string dbPath, IDbConfig dbConfig, ILogManager logManager, params T[] keys) : base(basePath, dbPath, dbConfig, logManager, GetColumnFamilies(keys))
+        protected ColumnsDb(string basePath, string dbPath, IDbConfig dbConfig, ILogManager logManager, string name, params T[] keys) : base(basePath, dbPath, dbConfig, logManager, GetColumnFamilies(dbConfig, name, GetEnumKeys(keys)))
         {
+            Name = name;
+            keys = GetEnumKeys(keys);
+
             foreach (var key in keys)
             {
                 _columnDbs[key] = new ColumnDb(Db, this, key.ToString()); 
             }
         }
 
-        private static ColumnFamilies GetColumnFamilies(T[] keys)
+        public override string Name { get; }
+
+        private static T[] GetEnumKeys(T[] keys)
+        {
+            if (typeof(T).IsEnum && keys.Length == 0)
+            {
+                keys = Enum.GetValues(typeof(T)).Cast<T>().ToArray();
+            }
+
+            return keys;
+        }
+
+        private static ColumnFamilies GetColumnFamilies(IDbConfig dbConfig, string name, T[] keys)
         {
             var result = new ColumnFamilies();
+            var blockCacheSize = ReadConfig<ulong>(dbConfig, nameof(dbConfig.BlockCacheSize), name);
             foreach (var key in keys)
             {
-                result.Add(key.ToString(), new ColumnFamilyOptions());
+                var columnFamilyOptions = new ColumnFamilyOptions();
+                columnFamilyOptions.OptimizeForPointLookup(blockCacheSize);
+                columnFamilyOptions.SetBlockBasedTableFactory(new BlockBasedTableOptions().SetFilterPolicy(BloomFilterPolicy.Create()));
```

Comment:
are default table options ok???

Reply:
I think so, they are similar to db options.